index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section></section>
```

Comment:
- [ ] If I may ask, please, what is this empty section for? it is causing an issue on the page so, I would suggest you remove it

<img width="382" alt="1" src="https://user-images.githubusercontent.com/77843050/212500898-c8f2895c-961a-4387-a1c8-915ae985b0ba.png">